src/client/pages/Geo/GeoMap/GeoMapMenuItem/geoMapMenuItem.tsx

```diff
@@ -10,6 +10,7 @@ interface Props {
   tabIndex: number
   checked?: boolean
   onCheckboxClick?: () => void
+  children?: React.ReactNode
```

Comment:
better no to add children as prop. but use as props type React.
```typescript 
const GeoMenuItem: React.FC<React.PropsWithChildren<Props>>
```

Reply:
Right, thanks!